ultravox/data/datasets.py

```diff
@@ -768,6 +803,7 @@ def __init__(self, args: VoiceDatasetArgs) -> None:
             )
         if self._args.shuffle:
             ds = ds.shuffle(seed=self._args.shuffle_seed)
+        ds = SizedIterableDataset(ds, 0)
```

Comment:
What's exactly the point of using 0 here?
Also, I thought `SizedIterableDataset` was supposed to be the parent class. Why use it both as a parent class and as a wrapper.